backend/iam/views.py

```
@@ -7,9 +10,9 @@
 from django.utils.decorators import method_decorator
 from django.utils.translation import gettext_lazy as _
 from django.views.decorators.csrf import ensure_csrf_cookie
-from knox.auth import TokenAuthentication
-from knox.views import LoginView as KnoxLoginView
-from rest_framework import permissions, serializers, status, views
+from knox.auth import TokenAuthentication, get_token_model, knox_settings
+from knox.views import DateTimeField, LoginView as KnoxLoginView, user_logged_in
```

Ruff (0.8.2)

`knox.views.user_logged_in` imported but unused

Remove unused import: `knox.views.user_logged_in`

(F401)